apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { useFormik } from 'formik';
+import { toast, TypeOptions } from 'react-toastify';
+import * as Yup from 'yup';
+
+import { EventRestriction } from '@mull/types';
+import { PillOptions, CustomTextInput, CustomTimePicker } from '@mull/ui-lib';
+import DateCalendar from '../create-event/date-calendar/date-calendar';
+
+import { DAY_IN_MILLISECONDS } from '../../../constants';
+
+import './create-event.scss';
+
+/* eslint-disable-next-line */
```

Comment:
The eslint line is just there when we generate components with Nx.

Reply:
Its because interfaces for props are already created for us with Nx, and they are initially empty. That line is there to avoid having tslint errors when you just generate components.